docs/querying/functions.md

```diff
@@ -272,24 +272,22 @@ For each timeseries in `v`, `label_join(v instant-vector, dst_label string, sepa
 using `separator` and returns the timeseries with the label `dst_label` containing the joined value.
 There can be any number of `src_labels` in this function.
 
-This example will return a vector with each time series having a `foo` label with the value `a,b,c` added to it:
+This example will return a vector with each time series having a `foo` label added to it, with the value `a,b,c`:
```

Comment:
This does not look like an improvement to me; rest LGTM

Reply:
I think I was objecting to the pronoun-antecedent mismatch ("it" refers to "time series" not "label"), but the whole sentence is awkward. Maybe this:
> This example will transform a vector of time series, by adding to each a label `foo` with value `a,b,c`

?

Reply:
It's the label which has the value added, so it's grammatically correct.

Reply:
I've reverted the line in question.